django_rest_generator/client.py

```diff
@@ -270,7 +270,9 @@ def _build_from_openapi_schema(self, schema_file: str = None):
 
     @classmethod
     def build_from_openapi_schema(cls, schema_file, token=None):
-        return cls(token=token)._build_from_openapi_schema(schema_file)
+        obj = cls(token=token)
+        obj._build_from_openapi_schema(schema_file)
+        return obj
```

Comment:
Another way to do this is to end `_build_from_openapi_schema` with `return self`. Mainly though, can you add docstrings?

Reply:
I think `return self` doesnt make sense from the private method
you create the object, build the schema, return the object
building the schema should return None, as its just a builder method.


i can see about adding doc strings, its weird to do for a method i didnt write

Reply:
Yeah, I get that. Maybe @moisesbenzan can help there?

It is a public method that needs to be overridden when using the client (looking at how we use it in weeblv2), so its important that its documented how it should be used.

Reply:
It doesn't need to be over written. its just being over written in our use case. from a general use case you should be OK to just use the the generic api client

See: https://github.com/canonical/django-rest-generator/pull/4/files